src/main/java/io/pravega/connectors/flink/sink/StreamNameGauge.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.connectors.flink.sink;
+
+import org.apache.flink.metrics.Gauge;
+
+/**
+ * Gauge for getting the fully qualified stream name information.
+ */
+public class StreamNameGauge implements Gauge<String> {
```

Comment:
As we do not have metrics support in this PR, this class can be removed.

Reply:
removed